docs/config-dir.rst

```diff
@@ -0,0 +1,121 @@
+    The first three are self explanatory. The **transition-tool** "socket" is used
+    by **t8ntool**, which runs a separate instance of **evm t8n** for each test.
+
+    You can find more information about the communication between **retesteth**
+    and clients in `the retesteth wiki 
+    <https://github.com/ethereum/retesteth/wiki/Add-client-configuration-to-Retesteth>`_.
+
+.. _socketAddress:
+
+  - **socketAddress**, the address of the socket, either a list of TCP ports (in 
+    the format **<ip>:<port>**), a file for IPC, or an executable to run (for
+    **transition-tool**).
+
+  - **initializeTime**, the time to wait for the client to initialize before
+    sending it tests.
+
+  - **forks**, the main supported forks.
+
+  - **additionalForks**, additional forks, which are supported but only if they 
+    are specified explicitly. For example, if a client's **config** file specifies:
+
+    ::
+
+      "forks" : [
+        "EIP158",
+        "Byzantium",
+        "Constantinople",
+        "ConstantinopleFix",
+        "Istanbul",
+        "Berlin"
+      ],
+      "additionalForks" : [
+        "EIP158ToByzantiumAt5",
+        "HomesteadToDaoAt5",
+        "ByzantiumToConstantinopleFixAt5"
+      ],        
+
+    And the test specifies **>=Byzantium**, it will test these forks:
+   
+    - Byzantium
+    - Constantinople
+    - ConstantinopleFix
+    - Istanbul
+    - Berlin
+
+    But not additional forks such as **ByzantiumToConstantinopleFixAt5**.
+    
+  - **exceptions**, the exception messages that the client emits for blocks that
+    are invalid in various ways. The key is the string used to identify the exception
+    in the **expectException** field of invalid block tests. The value is the message
+    the client emits.
+
+    .. note::
+
+       The exception is only checked if:
+
+       #. **-\\-filltests** is specified. 
+
+       #. The test is in **BlockchainTests/InvalidBlocks**.
+
+       Otherwise, either
+       **retesteth** only checks that an exception occured, not which exception it 
+       was (without **-\\-filltests**), or treats any exception as an abort (if the
+       test is not for invalid blocks).  
+
+
+- **start.sh**  the meaning of this script varies depending on the
+  method used to communicate with the client.
+
+  - With **tcp** and **ipc** clients the script
+    starts the client and possibly provides it with the port or pipe on 
+    which it should listen. In both cases it is possible to start multiple clients 
+    to run tests in parallel.
+
+    .. note::
+
+       If there is no **start.sh** script at all **retesteth** assumes that it 
+       needs to connect to an existing client rather than run its own.
+
+  - With **ipc-debug** clients the script is ignored, because it is assumed that the
+    that the client is already running in debug mode.
+
+  - With **transition-tool** clients the script is executed for every test, and the
```

Comment:
no the start.sh 
the script which is defined in socketAddress in a config file

Reply:
Fixed, thank you for noticing.